### ISSUE TITLE
DataCite enhance metadata -> Citations, relatedIdentifiers

### ISSUE THREAD

**Issue by tilfischer** [NFDI4Chem/nmrxiv#1027]
**Is your enhancement request related to a problem? Please describe.**
Users are allow to add citations for each project. Two use cases are to add the citation of 1) the corresponding article and 2) other corresponding datasets. 

After adding a citation (please also see #1026), user cannot selected a relatedIdentifier. For a dataset corresponding to an article, the relatedIdentifier should be IsSupplementTo and also for another dataset, where only the NMR data also got published again in nmrXiv, the relatedIdentifier could be IsSupplementTo.

**Describe the solution you'd like**
User need to be able to select a relatedIdentifer for a citation, which has a DOI. I agree that the list of relatedIdentifiers provided in the DataCite schema documentation is quite long and the whole list could be confusing to users. 

**Describe alternatives you've considered**
Instead of providing the full list of relatedIdentifiers, one could additional imagine a grouped list with the most often used relationTypes at the beginning of that list, possibly with a tooltip on more information on that. 

Best,
Tillmann

**Comment by NRayya** [comment]
Good afternoon @tilfischer 
In our implementation we already do what you suggest if I didn't misunderstand you, the DOI of the citation is provided as related identifier of type is supplemented to
<img width="431" alt="image" src="https://github.com/NFDI4Chem/nmrxiv/assets/82588017/622f4d24-db66-4306-af10-37a86fac7df0">

Please let me know if this solves the issue or if you need more details

**Comment by tilfischer** [comment]
This fits for adding a citation to a corresponding article. However, if another dataset is data this would not be the right relation type. There are numerous other use cases, hence, users should be able to select a relation type. 

As highlighted above, I would not provide the [long and confusing list](https://datacite-metadata-schema.readthedocs.io/en/4.5/appendices/appendix-1/relationType/#relationtype) but sort this somehow by grouping the most relevant ones (IsSupplementTo, IsSupplementeyBy, hasPart, IsPartOf) possibly with a tooltip for guidance.

**Comment by NRayya** [comment]
To ensure consistency, we are not allowing the user to pick the relation types while we are assigning them behind the scene.
We already offer has part and is part of to link the project, sample, spectra. And we offer IsSupplementTo for the citation. I think we only need to clarify the meaning of citation more in the UI. If you still believe there are important relations missing we can add the corresponding fields. Does this solve the issue?

**Comment by tilfischer** [comment]
Yes. No choice but a proper choice behind the scenes is also a good choice. An additional option could be to pick IsSupplementTo for a citation with a DOI registered with Crossref and can be validated against api.crossref.org (by assuming this is an article, as it was registered with Crossref) and pick hasPart for a DOI registered with DataCite and can be validated against api.datacite.org .

Best,
Tillmann

**Comment by NRayya** [comment]
Closed in favor of Datacite #949
